client/src/document/types.tsx

```diff
@@ -82,6 +82,15 @@ export interface MacroErrorMessage extends GenericFlaw {
   fixed?: true;
 }
 
+export interface TranslationDifferenceFlaw extends GenericFlaw {
+  difference: {
+    explanation: string;
+    type: string;
+    name: string;
+    fullExplanation: string[];
```

Comment:
wondering if there is a better name for it which captures the fact that it's **multiple** code-expressions

Reply:
`fullExplanations` (the `s` suffix) would be a good start.